src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeController.java

```diff
@@ -28,17 +28,17 @@ public EnvelopeController(
 
     @GetMapping
     @ApiOperation(
-        value = "Retrieves all envelopes for a given jurisdiction",
+        value = "Retrieves all envelopes with processed status for a given jurisdiction",
```

Comment:
We are not providing `status` as field in the response. Shall we rename to `Retrieve all processed envelopes for a given jurisdiction` instead?